lib/rubygems/commands/signout_command.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/commands/query_command'
+
+class Gem::Commands::SignoutCommand < Gem::Commands::QueryCommand
```

Comment:
um, you have point. I have copied code from another command file but forgot to change this portion of code. Thanks for point this out.